documentation/documentation/i18n/nl/docusaurus-plugin-content-docs/current/projectform/project_upload_form.md

```diff
@@ -0,0 +1,31 @@
+# Project upload handleiding
+Een leraar kan een nieuw project van zijn cursus uploaden door naar de url `projects/create` te gaan, waar hij gegevens moet invullen die overeenkomen met het project dat hij wil uploaden.
+
+![Projectformulier](/img/project_form_1.png)
+
+De eerste 3 velden zijn vrij eenvoudig, je hoeft alleen maar de titel in te vullen die je voor het project wilt, de beschrijving en de cursus te selecteren waarvoor je het project wilt. De cursusselectie bevindt zich in een vervolgkeuzemenu waarin je alle cursussen ziet waarvan je leraar bent.
```

Comment:
Same here, no need to say its simple

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/a175039634b28c72360c862b6f82fb2f88d1afe0